src/tribol/physics/CommonPlane.cpp

```diff
@@ -272,35 +273,17 @@ int ApplyNormal<COMMON_PLANE, PENALTY>( CouplingScheme* cs )
     auto numPolyVert = 2;
     // update if we are in 3d
     if ( dim == 3 ) {
-      auto& cp3 = static_cast<ContactPlane3D&>( plane );
-      numPolyVert = cp3.m_numPolyVert;
+      numPolyVert = plane.m_numPolyVert;
       xVert_size = 3 * numPolyVert;
     }
     initRealArray( xVert, xVert_size, 0. );
 
-    //      // get projected face coordinates
-    //      cpManager.getProjectedFaceCoords( cpID, 0, &xf1[0] ); // face 0 = first face
-    //      cpManager.getProjectedFaceCoords( cpID, 1, &xf2[0] ); // face 1 = second face
-
     // get current configuration, physical coordinates of each face
-    mesh1.getFaceCoords( index1, &xf1[0] );
-    mesh2.getFaceCoords( index2, &xf2[0] );
+    plane.getFace1Coords( &xf1[0] );
+    plane.getFace2Coords( &xf2[0] );
```

Comment:
```suggestion
    plane.getFace1Coords( xf1 );
    plane.getFace2Coords( xf2 );
```
These are already pointers

Reply:
This is a style/legacy thing that happens in a lot of places. I'll defer to a different PR to clean this up.